### PR TITLE
edit note with vim

### PR DESCRIPTION
**Pull request by djwgit**
edit note (today or a given date) with vim
- dnte
vim edit today's note
- dnoe yyyy-mm-dd
vim edit note for a given date, 
if no date given, vim netrw lists files in the ~/dn folder, could pick one to to edit. nice!

### CODE REVIEW

**Comment by tomlockwood** [comment]
Looks good! Can we maybe include a `dnoe` command which takes one argument, which is a date, and opens that dn file in the editor? Thanks!

**Comment by djwgit** [comment]
good idea, @tomlockwood
- added dnoe
- updated readme
ready for review...

**Review by tomlockwood — CHANGES_REQUESTED** [review]
LGTM with one proviso.  I'd prefer the commands are consistently documented in the README.  My suggestion is move the comment about `dnoe` allowing file selection into the README and delete the comments on both in `dn`.  Make that change and I'll merge!!!

Thanks!

File: dn [hunk 1]
```diff
@@ -8,11 +8,22 @@ dno() {
   echo " * $2" >> ~/dn/$1
 }
 
+# edit note for a given date, e.g. dnoe 2019-11-02
+# if no date given, vim lists the ~/dn folder, then pick one to edit
```
Comment (tomlockwood) [inline dn]: To make it clear what I'm talking about in my PR comment - I suggest removing this and the other command documentation here into the README.

**Comment by tomlockwood** [comment]
Awesome, thanks!

**Pull request merged**